tsconfig.json

```diff
@@ -3,8 +3,8 @@
         "typeRoots": [
             "node_modules/@types"
         ],
-        "module": "es6",
-        "moduleResolution": "node",
+        "module": "commonjs",
+        "moduleResolution": "Node",
```

Comment:
Sure about the capital 'n'?

Reply:
Does this mean we can never move to ES6, or does it mean that TS compiles to commonjs modules?